src/components/board.jsx

```diff
@@ -10,28 +11,32 @@ import Loadable from './loadable';
 import Progress from '../progress';
 import Database from '../database';
 
-const ProgressView = React.createClass({
-  getInitialState() {
-    return {message: null, ticks: 0, max: 0};
-  },
+class ProgressView extends Component {
+  state = {message: null, ticks: 0, max: 0};
+
   componentDidMount() {
     const {progress} = this.props;
     progress.on('start', this.onStart);
     progress.on('tick', this.onTick);
     progress.on('stop', this.onStop);
-  },
+  }
+
   componentWillUnmount() {
     const {progress} = this.props;
     progress.off('start', this.onStart);
     progress.off('tick', this.onTick);
     progress.off('stop', this.onStop);
-  },
-  onStart(context) { this.setState({max: this.props.progress.max, message: 'Start: ' + context}); },
-  onTick(context, ticks, max) {
+  }
+
+  onStart = (context) => { this.setState({max: this.props.progress.max, message: 'Start: ' + context}); };
+
+  onTick = (context, ticks, max) => {
     this.setState({ticks: ticks, max: max, message: context});
     this.forceUpdate();
-  },
-  onStop(context) { this.setState({ticks: this.props.progress.ticks, max: this.props.progress.max, message: 'Finished: ' + context}); },
+  };
+
+  onStop = (context) => { this.setState({ticks: this.props.progress.ticks, max: this.props.progress.max, message: 'Finished: ' + context}); };
```

Comment:
line is longer than github's code review block width.

drop `messages: ...` to next line under `ticks:` ?

or something similar.

Reply:
Please note that a 'split line' type of change should only be done in this PR where the change *already* contains a non-whitespace modification to the line.

We dont want `git diff -w` for this patch getting extra changes which are line splits.

There are only a few cases in this PR where the very long line is already changing syntax, so we may as well also improve the layout a little.